src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/idam/UserMapping.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.idam;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+import java.util.Map;
+
+@Component
+@ConfigurationProperties(prefix = "idam")
+public class UserMapping {
+    private Map<String, Credential> users;
```

Comment:
maybe rename to make it clear that the key in this map is service name.
`usersForService`, `serviceToCredentialsMapping`? Dunno. Or add a comment :P

Reply:
that would all go into the env variable then,
i'll just add a comment

Reply:
actually renamed the class